### PR TITLE
[6533] Update guidance pages to include placement content

### PR DESCRIPTION
**Pull request by darokel**
### Context

https://trello.com/c/zqiAOisd/6533-register-guidance-about-placements-2-updates-and-2-new-pages

### Changes proposed in this pull request

- Updates to existing guidance pages to include information about placements
- Adds two new guidance pages around placements

### Guidance to review

View the "how to use service" link and double check the new pages. The links are also below:

https://register-pr-3849.test.teacherservices.cloud/

### CODE REVIEW

File: app/views/guidance/bulk_upload_placement_data.html.erb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+      You can also use the Higher Education Statistics Agency (HESA) service to update trainees in-year, if you have access to it.
+    </p>
+
+    <h2 class="govuk-heading-m">How to bulk upload placement data</h2>
+
+    <ol class="govuk-list govuk-list--number">
+      <li>Make sure you are signed into Register. Click into ‘Bulk updates’ in the main navigation.</li>
+      <li>Download a CSV file listing the trainees with missing placement data.</li>
+      <li>Add missing placement URNs.</li>
+      <li>Upload the CSV file</li>
```
Comment (carlosmartinez) [inline app/views/guidance/bulk_upload_placement_data.html.erb:30]: being pedantic, should we be consistent as to whether these bullet points include a full stop at the end?

File: app/views/guidance/manage_placements.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    <p class="govuk-body-m">A minimum of 2 placements is required for each registered trainee with a successful course outcome in the <%= @previous_academic_cycle.label %> academic year.</p>
+
+    <p class="govuk-body-m">^ HEI providers will be notified by email in early <%= @performance_profile_sign_off_short_deadline %> when their placement data has been imported into Register for them to check and fill in any missing placement information. We will only import the URN from the student collection. </p>
+
+    <h3 class="govuk-heading-s">Manually record placement data</h3>
+
+    <p class="govuk-body-m">Manual input is recommended for providers with a small number of registered trainees.</p>
+
+    <p class="govuk-body-m">You must use manual input if you do not have school placement URNs</p>
```
Comment (carlosmartinez) [inline app/views/guidance/manage_placements.html.erb:36]:
```suggestion
    <p class="govuk-body-m">You must use manual input if you do not have school placement URNs.</p>
```

File: app/views/guidance/manage_placements.html.erb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+    <p class="govuk-body-m">Find out how to include placement data in trainee records manually.</p>
+
+    <% if FeatureService.enabled?(:bulk_placements) %>
+      <h3 class="govuk-heading-s">Bulk upload placement data</h3>
+
+      <p class="govuk-body-m">Bulk upload is recommended for those providers who have many registered trainees to manage.</p>
+
+      <p class="govuk-body-m">You can only use bulk upload if you have the school placement URN</p>
```
Comment (carlosmartinez) [inline app/views/guidance/manage_placements.html.erb:45]:
```suggestion
      <p class="govuk-body-m">You can only use bulk upload if you have the school placement URN.</p>
```

File: app/views/guidance/bulk_upload_placement_data.html.erb [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    </ol>
+
+    <h2 class="govuk-heading-m">1. Sign into Register</h2>
+
+    <p class="govuk-body-m">Once you have signed into Register, you can <%= govuk_link_to("access the bulk update feature", new_bulk_update_placements_path) %></p>
```
Comment (carlosmartinez) [inline app/views/guidance/bulk_upload_placement_data.html.erb:35]:
```suggestion
    <p class="govuk-body-m">Once you have signed into Register, you can <%= govuk_link_to("access the bulk update feature", new_bulk_update_placements_path) %>.</p>
```

File: app/views/guidance/manage_placements.html.erb [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+    <p class="govuk-body-m">This new requirement will be tied to trainees who studied in the <%= @previous_academic_cycle.label %> academic year for performance profiles sign off. </p>
+
+    <p class="govuk-body-m">The performance profile sign off period starts in <%= @performance_profile_sign_off_start_period %> with a deadline of <%= @performance_profile_sign_off_full_deadline %>.</p>
+
+    <h2 class="govuk-heading-m">What type of school placement data is required</h2>
+
+    <p class="govuk-body-m">You only need to provide location information about placements – the URN (if known) or the school name and postcode.</p>
+
+    <p class="govuk-body-m">^ You do not need to provide duration of time spent at each placement</p>
```
Comment (carlosmartinez) [inline app/views/guidance/manage_placements.html.erb:24]: is `^` doing something worthwhile here?

Comment (darokel) [reply app/views/guidance/manage_placements.html.erb:24]: not sure, I just copied all of this directly 🙈

**Review by carlosmartinez — APPROVED** [review]

**Comment by darokel** [comment]
[Performance profile sign off guidance - update](https://trello.com/c/RaPfi4fk/6547-performance-profile-sign-off-guidance-update)

File: app/views/guidance/bulk_upload_placement_data.html.erb (hunk 1 above)
Comment (darokel) [reply app/views/guidance/bulk_upload_placement_data.html.erb]: updated to be pedantic-ness compliant

File: app/views/guidance/manage_placements.html.erb (hunk 5 above)
Comment (darokel) [reply app/views/guidance/manage_placements.html.erb]: it's supposed to be an inset text, updated

**Pull request merged**